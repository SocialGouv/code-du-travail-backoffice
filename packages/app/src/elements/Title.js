import styled from "@emotion/styled";

const Title = styled.h1`
  color: black;
  font-size: 1.3rem;
  font-weight: 600;
  margin: 1rem 0;
  user-select: none;
`;

export default Title;
