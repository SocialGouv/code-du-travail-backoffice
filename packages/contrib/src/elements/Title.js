import styled from "styled-components";

export default styled.p`
  color: black;
  font-size: 1.25rem;
  font-weight: 600;
  margin: ${props => (props.isFirst ? "0 0 0.5rem" : "1.5rem 0 0.5rem")};
  user-select: none;
`;
