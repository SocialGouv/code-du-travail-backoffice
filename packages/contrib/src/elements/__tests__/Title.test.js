import React from "react";
import { render } from "react-testing-library";
import Title from "../Title";

describe("[Contrib] elements/<Title />", () => {
  it("should match snapshot", () => {
    const { container } = render(<Title />);

    expect(container).toMatchSnapshot();
  });

  it("should match snapshot with `isFirst` prop", () => {
    const { container } = render(<Title isFirst />);

    expect(container).toMatchSnapshot();
  });

  it("should match snapshot with `isSecondary` prop", () => {
    const { container } = render(<Title isSecondary />);

    expect(container).toMatchSnapshot();
  });
});
