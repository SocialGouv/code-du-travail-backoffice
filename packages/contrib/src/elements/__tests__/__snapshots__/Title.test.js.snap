// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`[Contrib] elements/<Title /> should match snapshot 1`] = `
<div>
  <p
    class="Title-r6qlw8-0 kDtypV"
  />
</div>
`;
